frame/alliance/src/mock.rs

```diff
@@ -83,8 +83,10 @@ impl pallet_balances::Config for Test {
 	type ReserveIdentifier = [u8; 8];
 }
 
+const MOTION_DURATION: u64 = 3;
```

Comment:
@muharem 
what is this duration 3?
duration_in_secs or duration_in_days or duration_in_blocks or duration_in_slots or duration_in_... ?
I would be more specific here

Reply:
Agreed, needs to be clear

Reply:
updated

Reply:
@muharem 
what about renaming these:
`MOTION_DURATION` -> `MOTION_DURATION_IN_BLOCKS`
`ALLIANCE_MOTION_DURATION` -> `ALLIANCE_MOTION_DURATION_IN_BLOCKS`
`MotionDuration` -> `MotionDurationInBlocks`
`AllianceMotionDuration` -> `AllianceMotionDurationInBlocks`

I guess, at least those constants should be renamed

I am checking other polkadot repos: `EPOCH_DURATION_IN_BLOCKS, EPOCH_DURATION_IN_SLOTS, slot_duration_millis, ACTIVE_DURATION_SECS, era_duration_millis`

Reply:
I think it's a bit outside of the scope of the purpose of this PR. 
And having a type as a suffix for variable name is controversial.